lib/grape/dsl/inside_route.rb

```diff
@@ -191,8 +191,12 @@ def body(value = nil)
       #
       #   GET /file # => "contents of file"
       def file(value = nil)
-        if value
-          @file = Grape::Util::FileResponse.new(value)
+        if value.is_a?(String)
+          file_body = Grape::ServeFile::FileBody.new(value)
+          @file = Grape::ServeFile::FileResponse.new(file_body)
+        elsif !value.is_a?(NilClass)
+          warn '[DEPRECATION] Argument as FileStreamer-like object will be deprecated. Use path to file instead'
```

Comment:
In other places we say "is deprecated", not "will be" and usually make this a full sentence with a period in the end. Like this: https://github.com/ruby-grape/grape/blob/4032ea8f7747483b84663c260d4aba0491bd7777/lib/grape/dsl/desc.rb#L50.